src/canvas/quadFeature.js

```diff
@@ -0,0 +1,123 @@
+//////////////////////////////////////////////////////////////////////////////
+/**
+ * Create a new instance of class quadFeature
+ *
+ * @class
+ * @param {Object} arg Options object
+ * @extends geo.quadFeature
+ * @returns {geo.canvas.quadFeature}
+ */
+//////////////////////////////////////////////////////////////////////////////
+geo.canvas.quadFeature = function (arg) {
+  'use strict';
+
+  if (!(this instanceof geo.canvas.quadFeature)) {
+    return new geo.canvas.quadFeature(arg);
+  }
+  geo.quadFeature.call(this, arg);
+
+  var m_this = this,
+      s_exit = this._exit,
+      s_init = this._init,
+      s_update = this._update,
+      m_quads;
+
+  ////////////////////////////////////////////////////////////////////////////
+  /**
+   * Build this feature
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  this._build = function () {
+    if (!m_this.position()) {
+      return;
+    }
+    m_quads = this._generateQuads();
+  };
```

Comment:
In `_build`.  It is only called when the quads change (added or removed), where as _renderImageQuads will get called on every render.